### PR TITLE
Do not sort array values while stringify. Fix tests.

### LINKED ISSUE #61: queryString.stringify sorts the values of the array breaking the app logic
As I do not see an option to reopen https://github.com/sindresorhus/query-string/issues/30 I'm filing this as a new issue. 

Please see my comment there https://github.com/sindresorhus/query-string/issues/30#issuecomment-221124798 which gives an example of why it is important to not sort the values of the array.

### PR DESCRIPTION
**Pull request by karanjthakkar**
Fixes #61. Thoughts @SamVerschueren?

### CODE REVIEW

File: test/stringify.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ test('handle array value', t => {
 	t.deepEqual(fn.stringify({abc: 'abc', foo: ['bar', 'baz']}), 'abc=abc&foo=bar&foo=baz');
 });
 
+test('should not sort array value', t => {
+	t.deepEqual(fn.stringify({abc: 'abc', foo: ['baz', 'bar']}), 'abc=abc&foo=baz&foo=bar');
```
Comment (SamVerschueren) [inline test/stringify.js]: `test('array order', () => {`

**Comment by SamVerschueren** [comment]
Besides the note, look good!

**Comment by karanjthakkar** [comment]
@SamVerschueren done!

**Comment by karanjthakkar** [comment]
@SamVerschueren bump!

**Comment by SamVerschueren** [comment]
@karanjthakkar It's looks good to me but I let @sindresorhus make the final decision. Thanks for the work :beers:!

**Pull request merged**